apps/web/src/lib/studies.ts

```diff
@@ -266,10 +266,30 @@ export const getStudiesForExport = async (organisationIds: number[]) => {
 
 export type UpdateStudyInput = Prisma.StudyUpdateInput
 
+export const mapCPMSStatusToSEStatus = (cpmsStatus: string): string => {
+  const statusMap: { [key: string]: string } = {
+    'Pre-Setup': 'In etup',
```

Comment:
Typo here.

I know there was some discussion around this, but using a hard coded string mapping here feels like a bad idea.

These strings could change in either system at any time and break the mapping, especially if they double up as display labels.

Reply:
updated spelling.

CPMS currently provides these status as strings. I'm not if there are any plans right now to assign ids @petecorriepa ?

Reply:
This is following the pattern of the ingest, I do like the idea of using db/id's - I'll make a note and we can pickup later, in the interests of getting this in the hands of Adam (who we only have for a limited time), I don't think we do as part of this ticket - ok with that @con-nihr ?